react-native/react-native-flipper/android/build.gradle

```diff
@@ -84,6 +84,14 @@ dependencies {
     debugImplementation("com.facebook.flipper:flipper:${FLIPPER_VERSION}") {
         exclude group:'com.facebook.fbjni'
     }
+
+    debugImplementation("com.facebook.flipper:flipper-network-plugin:${FLIPPER_VERSION}") {
```

Comment:
So I guess the trade-off here is that we make the setup a little simpler while forcing some more deps onto people that they may not want?

Reply:
I think we can add some properties to disable some plugins.
So, we add all plugins by default and we can use properties to disable some plugins, example: `FlipperDisablePlugin_Network`
and then use in Java [like this](https://github.com/react-native-async-storage/async-storage/blob/master/android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStoragePackage.java#L27)